frontend/src/components/InputField.js

```diff
@@ -1,11 +1,27 @@
 import React from "react";
+import store from "../reducers/index";
 
-export const InputField = ({ inputName, inputType, inputPh }) => {
+export const InputField = ({
+  inputName,
+  inputType,
+  inputPh,
+  inputValue,
+  onChangeFn,
+  onChangeFld,
+}) => {
   return (
     <div className="field">
       <label className="label">{inputName}</label>
       <div className="control">
-        <input className="input" type={inputType} placeholder={inputPh} />
+        <input
+          className="input"
+          type={inputType}
+          placeholder={inputPh}
+          onChange={(e) => {
```

Comment:
Why would we want to trigger a re-render on every keypress? Is there a reason we can't just update state on submit?